testdata/base64-gzip.tmx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<map version="1.2" tiledversion="1.2.2" orientation="orthogonal" renderorder="right-down" width="32" height="32" tilewidth="8" tileheight="8" infinite="0" nextlayerid="3" nextobjectid="2">
+<map version="1.2" tiledversion="1.2.2" orientation="orthogonal" renderorder="right-down" width="32" height="32" tilewidth="8" tileheight="8" infinite="false" nextlayerid="3" nextobjectid="2">
```

Comment:
The property in the TMX files is written as an `int` by Tiled, so we should leave them as 0's or 1's, could you revert the changes on the TMX files please